docs/reference/default-theme-config.md

```diff
@@ -1,6 +1,6 @@
 # Default Theme Config
 
-Theme config lets you customize your theme. You can define theme config via the `themeConfig` option in the config file:
+Theme config lets you customize your theme. You can define theme config via the `themeConfig` option inside [`.vitepress/config.[extension]`](./site-config):
```

Comment:
don't think this is needed either

Reply:
Perhaps it's obvious once you've used VitePress for a while, but as a newcomer it's not clear which config file this is referring to and where it's located. I initially wondered "_Does VitePress have more than one config file?_" and "_Is there a theme config file, or is it the same as the VitePress config file?_" and "_Which folder should I be looking in?_".

I think having the path here reduces that initial confusion.